src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -48,7 +48,7 @@
 
         <service id="file_locator" class="Symfony\Component\HttpKernel\Config\FileLocator">
             <argument type="service" id="kernel" />
-            <argument>%kernel.root_dir%/Resources</argument>
+            <argument>%kernel.root_dir%/../templates</argument>
```

Comment:
The fact it is  `%kernel.root_dir%/Resources` and not `%kernel.root_dir%/Resources/views` hints that `FileLocator` is not only about locating templates but any resource (see class and interface description), replacing this path by `templates` does not look right to me.
Also making `FileLocator` suddenly stop being able to locate resources from the deprecated path is not good regarding BC, using it for such resources should trigger a deprecation notice first. 
We will probably need to deprecate this argument at the class level before 5.0 since there is no global `Resources` directory anymore (directories previously stored in the `app/Resources` folder are now first-level dirs like `/templates` and `/translations`), and `kernel.root_dir` won't exist anymore in 5.0.